spacelift/internal/structs/scheduled_task.go

```diff
@@ -0,0 +1,31 @@
+package structs
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/shurcooL/graphql"
+)
+
+// ScheduledTask represents a scheduled task.
+type ScheduledTask struct {
```

Comment:
I believe we would want to support task that are scheduled for given timestamp as well